statsmodels/stats/diagnostic.py

```diff
@@ -1,14 +1,103 @@
-#collect some imports of verified (at least one example) functions
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from scipy import stats
+
+# collect some imports of verified (at least one example) functions
 from statsmodels.sandbox.stats.diagnostic import (
     acorr_ljungbox, breaks_cusumolsresid, breaks_hansen, CompareCox, CompareJ,
     compare_cox, compare_j, het_breuschpagan, HetGoldfeldQuandt,
     het_goldfeldquandt, het_arch,
-    het_white, recursive_olsresiduals, acorr_breusch_godfrey,
+    recursive_olsresiduals, acorr_breusch_godfrey,
     linear_harvey_collier, linear_rainbow, linear_lm,
     unitroot_adf,
     het_breushpagan, acorr_breush_godfrey  # deprecated because of misspelling
     )
 
-from ._lilliefors import (kstest_fit, lilliefors, lillifors, kstest_normal,
-                          kstest_exponential)  # lillifors is deprecated
-from ._adnorm import normal_ad
+from ._lilliefors import (  # noqa:F401
+    kstest_fit, lilliefors, lillifors, kstest_normal,
+    kstest_exponential)  # lillifors is deprecated
+from ._adnorm import normal_ad  # noqa:F401
+
+
+# -----------------------------------------------------------------
+# Misc Helpers
+
+class ResultsStore(object):
```

Comment:
Where did this come from?

Reply:
Top of the sandbox file.  For now copied instead of moved.